distribution/lib/Standard/Base/0.0.0-dev/src/Data/Filter_Condition.enso

```diff
@@ -199,6 +199,38 @@ type Filter_Condition
         Is_In values -> values.contains
         Not_In values -> elem -> values.contains elem . not
 
+    ## PRIVATE
+       Convert to a display representation of this Filter_Condition.
+    to_display_text : Text
+    to_display_text self =
+        condition = case self of
+            Less value -> "<" + value.to_display_text
+            Equal_Or_Less value -> "<=" + value.to_display_text
+            Equal value -> "==" + value.to_display_text
+            Equal_Or_Greater value -> ">=" + value.to_display_text
+            Greater value -> ">" + value.to_display_text
+            Not_Equal value -> "!=" + value.to_display_text
+            Between lower upper -> "Between " + lower.to_display_text + " And " + upper.to_display_text
+            Starts_With prefix case_sensitivity ->
+                "Starts With " + prefix.to_display_text + (if case_sensitivity == Case_Sensitivity.Default then "" else " Case " + case_sensitivity.to_display_text)
```

Comment:
I'd extract this into a local helper at the top of this function:
```
render_case case_sensitivity =
    if case_sensitivity == Case_Sensitivity.Default then "" else " Case " + case_sensitivity.to_display_text
```
and then do
```suggestion
                "Starts With " + prefix.to_display_text + (render_case case_sensitivity)
```

I think it will be cleaner than repeating this 4 times.